server/src/assembly/resources/conf/iotdb-datanode.properties

```diff
@@ -17,6 +17,14 @@
 # under the License.
 #
 
+####################
+### Cluster Configuration
+####################
+
+# Used for indicate cluster name and distinguish different cluster.
+# Datatype: string
+cluster_name=defaultCluster
```

Comment:
Both iotdb-datanode.properties and iotdb-confignode.properties have this parameter, which violates our [design](https://apache-iotdb.feishu.cn/docx/doxcnjzB2xd9sScIlkNYJUJA1zb) about the configuration files. Move it to iotdb-common.properties instead.

Reply:
Fixed~